server/clients/index.js

```diff
@@ -40,4 +40,4 @@ module.exports = {
     infoService,
     templatesService,
     documentService
-};
\ No newline at end of file
+};
```

Comment:
Ideally these changes would be in a separate commit. (And you'd rebase away the intermediate commit that does nothing, I guess?)

Reply:
I don't actually know where these changes are from, I'd rebased away the commit that does nothing and although the files appear in a `git show --word-diff` there is no change shown. IntelliJ also doesn't track a change here with its Git Blame annotations.

Reply:
It's not too important; it's not worth blocking a merge over.